### PR TITLE
Fix nss-resolve reverse alias lookups

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by bluca — APPROVED** [review]
lgtm

File: src/nss-resolve/nss-resolve.c [hunk 1]
```diff
@@ -540,8 +540,8 @@ static void name_parameters_destroy(NameParameters *p) {
 }
 
 static const JsonDispatch name_parameters_dispatch_table[] = {
-        { "ifindex", JSON_VARIANT_INTEGER,  json_dispatch_ifindex, offsetof(NameParameters, ifindex), 0              },
-        { "name",    JSON_VARIANT_UNSIGNED, json_dispatch_string,  offsetof(NameParameters, name),    JSON_MANDATORY },
+        { "ifindex", JSON_VARIANT_INTEGER, json_dispatch_ifindex, offsetof(NameParameters, ifindex), 0              },
+        { "name",    JSON_VARIANT_STRING,  json_dispatch_string,  offsetof(NameParameters, name),    JSON_MANDATORY },
```
Comment (poettering) [inline src/nss-resolve/nss-resolve.c:543]: ouch ouch ouch

**Review by poettering — APPROVED** [review]
Not too much of a fan of moving the variable declarations into the function bodies. I kinda like the separation of variable declarations from code I must say. but ok.

**Comment by bluca** [comment]
Usual suspects in the CI - merging

**Pull request merged**